Source/WebKit/UIProcess/Inspector/mac/WKInspectorViewController.mm

```diff
@@ -176,7 +178,18 @@ - (WKWebViewConfiguration *)webViewConfiguration
     // Ensure that a page group identifier is set. This is for computing inspection levels.
     if (!configuration.get()._groupIdentifier)
         [configuration _setGroupIdentifier:WebKit::defaultInspectorPageGroupIdentifierForPage(_inspectedPage.get())];
-    
+
+    // Prefer using a custom persistent data store if one exists.
+    WKWebsiteDataStore *targetDataStore = nil;
```

Comment:
This should be `RetainPtr`

Reply:
No `= nil` when you switch to `RetainPtr<WKWebsiteDataStore>`